3.10/analyzers.md

```diff
@@ -1264,10 +1264,10 @@ result can be used with. For example the *text* type will produce
 
 Currently the following *features* are supported:
 
-- **frequency**: how often a term is seen, required for `PHRASE()`
-- **norm**: the field normalization factor
-- **position**: sequentially increasing term position, required for `PHRASE()`.
-  If present then the *frequency* feature is also required
+- **frequency**: track how often a term occurs. Required for `PHRASE()`, `BM25()`, `TDIDF()`.
+- **norm**: normalize repeated terms to score fairer. Recommended for `BM25()` and `TFIDF()`.
```

Comment:
As a separate comment: `normalize repeated terms to score fairer` is not entirely correct. Suggest to keep original description (and just add "Required for" remark)

Reply:
> it is required for BM25 with default settings

I [noted down](https://github.com/arangodb/docs/issues/222) the following from a conversation with Andrey:
For BM25 it is not meaningful to disable norm [...] Recommended to use it - "not meaningful" would mean possible, but the results would possibly be unexpected or even useless. Is it required for a true BM25 scoring but not for BM11 or BM15, or both?

About the norm feature, I noted down:
Evens out scoring to make things fairer, so that ["bar","bar","bar"] vs. "bar" get more similar or even identical values - I tried to convey that in a succinct way and find it easier to understand than "field normalization factor", which sounds like the name for an implementation detail that doesn't really explain what it is for. Do you have an alternative suggestion for describing it?

Reply:
>Is it required for a true BM25 scoring but not for BM11 or BM15, or both?
Only BM15 will run without `norm`
> Do you have an alternative suggestion for describing it
Ok, I've got the point. Let's combine it and make every side happy.
"Writes the field length normalization factor.  Used to score repeated terms fairer"